workflow-controller/controller-service/src/main/java/io/hyscale/controller/commands/args/FileConverter.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2019 Pramati Prism, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.hyscale.controller.commands.args;
+
+import java.io.File;
+
+import org.apache.commons.lang3.StringUtils;
+
+import io.hyscale.commons.utils.WindowsUtil;
+import picocli.CommandLine.ITypeConverter;
+
+public class FileConverter implements ITypeConverter<File> {
+    
+    /**
+     * Converts user input filepath to unix if host is windows
+     */
+    @Override
+    public File convert(String filePath) throws Exception {
+        
+        if (StringUtils.isBlank(filePath)) {
+            return null;
+        }
+        return WindowsUtil.isHostWindows() ? new File(WindowsUtil.updateToUnixFileSeparator(filePath))
+                : new File(filePath);
+    }
+
```

Comment:
if WindowsUtil.isHostWindows(){
     filepath = WindowsUtil.updateToUnixFileSeparator(filepath);
}
return new File(filepath);

Without Creating objects on ternary operators 
I think this is more readable